modules/packages/TOML.chpl

```diff
@@ -753,20 +753,28 @@ used to recursively hold tables and respective values
       return false;
     }
 
-    proc set(tbl: string, toml: unmanaged Toml?) {
+    proc set(tbl: string, toml: shared Toml?) {
+      ref t = this(tbl);
+      if t == nil {
+        t = toml;
+      } else {
+        t = toml;
+      }
+    }
+
+    proc set(tbl: string, toml: Toml) {
```

Comment:
I'm wondering why we have this overload if we are expecting them all to be `shared` now. Was there a specific test failure this addressed? If we stick with this one, can we drop the `nil` check since the formal isn't a `nilable toml` anymore?

Reply:
Since this is generic, I believe that it can still be `nilable`, but I think we can still drop that nil check since it wasn't doing anything.

Reply:
Nevermind, I was wrong, `nilable` objects cannot be passed to this function anymore, but that shouldn't matter. Anywhere that is calling it with a nilable can just call it with the `!` operator (where there shouldn't be any instances today, but will let a paratest prove that before saying it definitively).